### PR TITLE
Type spacing updates

### PR DESCRIPTION
**Pull request by Scotchester**
Updates for our latest type spacing guidelines.
## Additions
- New copy and images for type spacing section.
## Removals
- Images that are no longer needed
## Changes
- Change all instances of "header" to "heading"
- Rename Web "subheading" to "lead paragraph"
## Testing
- http://scotchester.github.io/design-manual/identity/typography.html#spacing
## Review
- @nataliafitzgerald 
- @jimmynotjim 
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [x] Passes all existing automated tests
- [x] New functions include new tests
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged
- [x] Visually tested in supported browsers and devices 
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: identity/typography.md [hunk 1]
```diff
@@ -121,9 +121,9 @@ abcdefghijklmnopqrstuvwxyz
 
 <p>Hierarchy refers to the difference in type size and weight between text elements. It creates focus points that signal the user where to read; a successful hierarchy enables readers to easily scan content.</p>
 
-### Headers
+### Headings
 
-<p>Consistent scaling, weights, and capitalization are used to create distinction between header levels. They provide the user with a familiar focus point when they scan through text and it helps organize content. Headers are used in three weights of Avenir Next Demi Bold, Medium, and Regular.</p>
+<p>Consistent scaling, weights, and capitalization are used to create distinction between heading levels. They provide the user with a familiar focus point when they scan through text and it helps organize content. Headings are used in three weights of Avenir Next Demi Bold, Medium, and Regular.</p>
```
Comment (jimmynotjim) [inline identity/typography.md:126]: You don't need `p` tags here do you?

**Comment by jimmynotjim** [comment]
One small comment, otherwise :+1:

**Comment by jimmynotjim** [comment]
Whoops, running this locally and "Display Heading" and "Lead Paragraph" aren't rendering

screen shot 2015-11-23 at 4 25 31 pm

screen shot 2015-11-23 at 4 25 57 pm

**Comment by Scotchester** [comment]
Have you installed the latest cf-core?

**Comment by jimmynotjim** [comment]
Ha, you'd think I'd know that seeing how I made that point last week. Must be Monday. Looks good :+1: from me again.

File: identity/typography.md (hunk 1 above)
Comment (Scotchester) [inline identity/typography.md:126]: Technically, no, but they're all over, and they don't do any harm, so I'll just leave 'em.

**Comment by jimmynotjim** [comment]
@cfarm @KimberlyMunoz @ascott1 anyone else want to sign off on this?

**Comment by Scotchester** [comment]
@nataliafitzgerald gave me a verbal thumbs-up to this last week.

**Pull request merged**